### PR TITLE
ClearLine Optimization

### PR DESCRIPTION
**Pull request by moofoo**
It is only necessary to call `this.stream.clearLine()` when `clear` is called in the `stop` method, otherwise clearing just to the right of the cursor (with `this.stream.clearLine(1)`) is sufficient. (See: https://stackoverflow.com/a/59805130)

While that might not seem like a big change, this resolves flickering issues in Windows that occur with many 'emulated' (if that's the correct term) shells I tested (See: sindresorhus#140). That includes WSL, all shells running under Windows Terminal, all shells running under Terminus, Powershell under Alacritty (which uses an OpenGL renderer), among others. No idea at all what the common thread could be. Plain old Powershell has no flickering issues, vanilla CMD stutters sometimes but is largely fine.  Cmder stands out as having no flickering issues, at least with the shells I tested. 

In any case, a call to `this.stream.clearLine(-1)` when indentation changes (to be greater) is needed, otherwise no other changes seem to be necessary. All tests pass.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -259,7 +263,7 @@ class Ora {
 		return fullPrefixText + frame + fullText;
 	}
 
-	clear() {
+	clear(stopping = false) {
```
Comment (sindresorhus) [inline index.js:266]: You should not expose this implementation detail to the user.

Comment (sindresorhus) [inline index.js:266]:
```suggestion
	clear(isStopping = false) {
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
While this change does pass existing tests, changes to the rendering come with some risk and I would feel more at ease with it if you added some additional tests.

File: index.js (hunk 1 above)
Comment (moofoo) [reply index.js]: Oh, I see what you mean, that's a public method. Hmm.

**Comment by moofoo** [comment]
> While this change does pass existing tests, changes to the rendering come with some risk and I would feel more at ease with it if you added some additional tests.

Rodger

**Comment by moofoo** [comment]
Seeing how much time has elapsed and how my new solution w/ tests is completely different than this one, and also I'm lazy when it comes to git, I'm just going to close this pull request and start a new one.

**Pull request closed without merging**